src/pcap/cmd/mock-bosh-api/main.go

```diff
@@ -0,0 +1,158 @@
+package main
```

Comment:
Consider moving this file into the test package, as it is not to be used for the actual release.

Reply:
will keep for now, will be removed before release along with agent-cli and api-cli